kroma-chain-ops/immutables/immutables.go

```diff
@@ -48,7 +48,7 @@ type PredeploysImmutableConfig struct {
 	}
 	L1BlockNumber  struct{}
 	GasPriceOracle struct{}
-	L1Block        struct{}
+	KromaL1Block   struct{}
```

Comment:
This kind of changes are for getting the original `L1Block` contract(now `KromaL1Block`) code from kroma-bindings.